the-list-be/server.js

```diff
@@ -1,19 +1,39 @@
 import Koa from 'koa';
 import Router from '@koa/router';
+import { MongoClient } from 'mongodb';
+import koaBody from 'koa-body';
 
 const app = new Koa();
 const router = new Router();
 
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri);
+const db = client.db('database');
+
+const students = db.collection('students');
+
+router.post('/student', koaBody(), async ctx => {
```

Comment:
`koaBody` здесь не нужен, т.к. мы его подключили в `app.use` ниже.
```suggestion
router.post('/student', async ctx => {
```